notebooks/README.md

```diff
@@ -149,6 +149,8 @@ More amazing notebooks here!
 | [225-stable-diffusion-text-to-image](225-stable-diffusion-text-to-image)<br> | Text-to-image generation with Stable Diffusion method | <img src=https://user-images.githubusercontent.com/15709723/200945747-1c584e5c-b3f2-4e43-b1c1-e35fd6edc2c3.png width=225>|
 | [226-yolov7-optimization](226-yolov7-optimization/)<br> | Optimize YOLOv7 using NNCF PTQ API | <img src=https://raw.githubusercontent.com/WongKinYiu/yolov7/main/figure/horses_prediction.jpg  width=225> |
 | [227-whisper-subtitles-generation](227-whisper-subtitles-generation/)<br> | Generate subtitles for video with OpenAI Whisper and OpenVINO | <img src=https://user-images.githubusercontent.com/29454499/204548693-1304ef33-c790-490d-8a8b-d5766acb6254.png  width=225> |
+| [228-clip-zero-shot-image-classification](228-clip-zero-shot-image-classification)<br> | Perform Zero-shot Image Classification with CLIP and OpenVINO | <img src=https://user-images.githubusercontent.com/29454499/207795060-437b42f9-e801-4332-a91f-cc26471e5ba2.png  width=225> |
```

Comment:
Could we zoom in on the thumbnail (graph) to show that that is a dog?

Reply:
what do you mean? Should change resolution of image?

Reply:
Updated